osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -103,6 +102,14 @@ private void load()
             }
         }
 
+        protected override void SkinChanged(ISkinSource skin, bool allowFallback)
+        {
+            base.SkinChanged(skin, allowFallback);
+
+            if (HitObject is IHasComboInformation combo)
+                AccentColour = skin.GetColour($"Play/Combo/{combo.ComboIndex}") ?? Color4.White;

```

Comment:
This shouldn't be at a `DrawableHitObject` level. Should be re-implemented everywhere it's used for accent colours specifically.

Imagine a hitobject that uses `IHasComboInformation` but doesn't modify colour - e.g. maybe displays a different animation on the last hitobject in a combo.

Reply:
can't we just ignore AccentColour in that case?

Reply:
And if we still use `AccentColour` to colorise the notes? E.g. mania notes, but maybe having like a super-white-colored note as the last note denoting the end of a combo.